staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -196,6 +196,18 @@ var (
 		[]string{"verb", "resource", "subresource"},
 	)
 
+	requestFilterLatency = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Name: "apiserver_request_filter_duration",
+			Help: "How much time the request spends in apf",
+			Buckets: []float64{0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0, 15.0, 20.0, 25.0, 30.0,
```

Comment:
My first thought: measure the .1-50ms region with high resolution, and then use exponential buckets up to 60s (4x the in-practice max with default flags).